src/site/xdoc/server/packaging.xml

```diff
@@ -0,0 +1,198 @@
+<?xml version="1.0"?>
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one
+  or more contributor license agreements.  See the NOTICE file
+  distributed with this work for additional information
+  regarding copyright ownership.  The ASF licenses this file
+  to you under the Apache License, Version 2.0 (the
+  "License"); you may not use this file except in compliance
+  with the License.  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing,
+  software distributed under the License is distributed on an
+  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  KIND, either express or implied.  See the License for the
+  specific language governing permissions and limitations
+  under the License.
+-->
+<document>
+
+    <properties>
+        <title>Apache James Server 3 - Packaging</title>
+    </properties>
+
+    <body>
+
+        <section name="Packaging available">
+            <p>James components are assembled in various way, called packaging.</p>
+
+            <ul>The packaging you choose will influence:
+                <li>Your java requirement</li>
+                <li>The protocol you supports</li>
```

Comment:
who is "you" ? it's weird, it's probably more the server that supports protocols, isn't it ?